### PR TITLE
Explicit build scripts

### PR DESCRIPTION
**Pull request by bollwyvl**
This is an attempt at #8... well, it builds locally.

### CODE REVIEW

**Comment by bollwyvl** [comment]
- win32/27
  - markupsafe. speedups.
  - It's known.
  - dropping from matrix, to be revisited

- linux32/27 https://anaconda.org/anaconda-nb-extensions/nb_conda/builds/6/0#L171
 - heh, this issue is about windows :trollface:

File: conda.recipe/post-link.bat [hunk 1]
```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
+"%PREFIX%\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
```
Comment (damianavila) [inline conda.recipe/post-link.bat]: we should use %PYTHON% here...

Comment (bollwyvl) [inline conda.recipe/post-link.bat]: That's what i changed it from, as it couldn't be found...

**Comment by bollwyvl** [comment]
I don't know how I had missed `run_test.py`, but that is probably the way of justice... if that test runs properly, we should probably use that approach wherever possible... on nbpresent, though, there would be a bunch of subprocess nonsense, so i'll keep the bat/sh files.

File: conda.recipe/post-link.bat (hunk 1 above)
Comment (damianavila) [inline conda.recipe/post-link.bat]: mmm... weird...

File: conda.recipe/run_test.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import nose
+import nb_conda
+
+nose.main(nb_conda)
```
Comment (damianavila) [inline conda.recipe/run_test.py]: This also need `%PREFIX%/python` instead of `%PYTHON%`

File: .binstar.yml [hunk 3]
```diff
@@ -19,3 +19,11 @@ script:
   - conda build --quiet conda.recipe
 
 build_targets: conda
+
+---
+
+platform:
+  - win-32
```
Comment (damianavila) [inline .binstar.yml]: Did you open an issue for the win 32 failure... linux 32 issue should be solved in the next Anaconda Repo release... I think

File: conda.recipe/post-link.bat (hunk 1 above)
Comment (damianavila) [inline conda.recipe/post-link.bat]: Should not be `"%PREFIX%\python.exe"`... btw, how it is possible that the python exe on win is not living in Scripts and lives in prefix???

File: conda.recipe/pre-unlink.bat [hunk 4]
```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_conda.setup remove --prefix="%PREFIX%"
+"%PREFIX%\python" -m nb_conda.setup remove --prefix="%PREFIX%"
```
Comment (damianavila) [inline conda.recipe/pre-unlink.bat]: same here, python.exe

File: conda.recipe/run_test.py (hunk 2 above)
Comment (damianavila) [inline conda.recipe/run_test.py]: forget about this...

**Comment by damianavila** [comment]
@bollwyvl just the `.exe`-related comment... I tried win32/27 and linux32/27

Hopefully the linux one will be solved this week with the new Repo release

@bollwyvl about the win one, did you reported it somewhere?

```
4An unexpected error has occurred, please consider sending the
375following traceback to the conda GitHub issue tracker at:
376
377    https://github.com/conda/conda-build/issues
378
379Include the output of the command 'conda info' in your report.
380
381
382Traceback (most recent call last):
383  File "C:\Users\binstar\Miniconda\Scripts\conda-build-script.py", line 4, in <module>
384    sys.exit(main())
385  File "C:\Users\binstar\Miniconda\lib\site-packages\conda_build\main_build.py", line 190, in main
386    args_func(args, p)
387  File "C:\Users\binstar\Miniconda\lib\site-packages\conda_build\main_build.py", line 468, in args_func
388    args.func(args, p)
389  File "C:\Users\binstar\Miniconda\lib\site-packages\conda_build\main_build.py", line 455, in execute
390    channel_urls=channel_urls, override_channels=args.override_channels)
391  File "C:\Users\binstar\Miniconda\lib\site-packages\conda_build\build.py", line 540, in test
392    channel_urls=channel_urls, override_channels=override_channels)
393  File "C:\Users\binstar\Miniconda\lib\site-packages\conda_build\build.py", line 285, in create_env
394    plan.execute_actions(actions, index, verbose=verbose)
395  File "C:\Users\binstar\Miniconda\lib\site-packages\conda\plan.py", line 537, in execute_actions
396    inst.execute_instructions(plan, index, verbose)
397  File "C:\Users\binstar\Miniconda\lib\site-packages\conda\instructions.py", line 148, in execute_instructions
398    cmd(state, arg)
399  File "C:\Users\binstar\Miniconda\lib\site-packages\conda\instructions.py", line 62, in EXTRACT_CMD
400    install.extract(config.pkgs_dirs[0], arg)
401  File "C:\Users\binstar\Miniconda\lib\site-packages\conda\install.py", line 484, in extract
402    t.extractall(path=path)
403  File "C:\Users\binstar\Miniconda\lib\tarfile.py", line 2078, in extractall
404    self.extract(tarinfo, path)
405  File "C:\Users\binstar\Miniconda\lib\tarfile.py", line 2115, in extract
406    self._extract_member(tarinfo, os.path.join(path, tarinfo.name))
407  File "C:\Users\binstar\Miniconda\lib\tarfile.py", line 2191, in _extract_member
408    self.makefile(tarinfo, targetpath)
409  File "C:\Users\binstar\Miniconda\lib\tarfile.py", line 2231, in makefile
410    with bltn_open(targetpath, "wb") as target:
411IOError: [Errno 13] Permission denied: u'C:\\Users\\binstar\\Miniconda\\pkgs\\markupsafe-0.23-py27_0\\Lib\\site-packages\\markupsafe\\_speedups.pyd'
```

**Comment by damianavila** [comment]
Merging this one, I will fix the exe and enable the matrix in another PR

**Pull request merged**